docs/application/vscode-ext/web.md

```diff
@@ -12,64 +12,37 @@ Before installing VS Code Extension for Tizen Web, check [Prerequisites of Visua
-4. After the installation is complete, click **Reload** to activate the extension.
-5. Once the extension activates, install the SDK to effectively use the extension.
-6. Open **Command Palette** and enter **Create Wizard Project**. The confirmation message dialog appears:
-
-   ![Visual Studio Code Marketplace](media/tizen_web1.png)
-
-   > **Note**
-   >
-   > Extension is a convenient way to create a project. It asks whether you want to use an existing Tizen baseline SDK installation or install a new one.
-
-  - To perform a fresh installation, click **YES**.
-
-    To perform a fresh installation and to begin the Tizen Baseline SDK installation, click YES. Else, click No, if you have already done the installation.
+2. In the **EXTENSIONS** view that appears, click the search box at the top and type **Tizen Web**.
+3. In the **Tizen Web** search result that appears, click **Install**.
+4. After the installation is complete, to activate the extension, click **Reload**.
 
-    > **Note**  
-    >
-    > In case you click **No**, it is assumed that the installation is done and the project is already created.
+    After the extension is active, **Tizen Web** installation is complete.
 
-## Install using VSIX file
+## Install using VSIX File
 
-To install VS Code Extension for Tizen Web using a VSIX file, follow these steps:
+To install VS Code extension for Tizen Web from a VSIX file, follow these steps:
 
-1. In the VS Code at the bottom of the **Activity Bar**, click **Extensions** (![extension icon](media/extensions.png)).
-2. Click **More** (![More icon](media/vscode_more_icon.png)) in the top-right corner and select **Install from VSIX**.
+1. In the Activity Bar of VS Code, click **Extensions** ![extension icon](media/extensions.png).
+2. In the **EXTENSIONS** view that appears, click **More** ![More icon](media/vscode_more_icon.png) > **Install from VSIX**.
 
    ![Install from VSIX](media/tizen_web2.png)
 
-3. Select an appropriate VSIX file in the file browser and click **Install**.
+3. In the **Install from VSIX** window that appears, select an appropriate VSIX file and click **Install**.
 4. After the installation is complete, to activate the extension, click **Reload**.
-
-   ![Install from VSIX](media/tizen_web3.png)
-
-5. After the extension activates, open **Command Palette** and enter **Create Wizard Project**. Choose, whether you want to use an existing Tizen Baseline SDK installation or install a new one:
-
-   ![Baseline SDK installation query](media/tizen_web5.png)
-
-   - To perform a fresh installation, click **YES**.
-
-     To perform a fresh installation and to begin the Tizen Baseline SDK installation, click **YES**. Else, click **No**.
-     > **Note** 
-     >
-     > In case you click **No**, it is assumed that the installation is done and the project is already created.
+5. After the extension is active, **Tizen Web** installation is complete.
```

Comment:
Please remove this as step 5.

It is a step result for step 4:

4. After the installation is complete, to activate the extension, click **Reload**.
    After the extension is active, **Tizen Web** installation is complete.

Reply:
done.